### PR TITLE
EPMDJ-7717 Added method for clearing thread local in plugin

### PR DESCRIPTION
**Pull request by ViktoriiaLebedeva**

### CODE REVIEW

File: src/jvmMain/kotlin/com/epam/drill/agent/instrument/TomcatTransformer.kt [hunk 1]
```diff
@@ -37,10 +37,13 @@ actual object TomcatTransformer {
             ClassPool.getDefault().appendClassPath(LoaderClassPath(loader as? ClassLoader))
             ClassPool.getDefault().makeClass(ByteArrayInputStream(classFileBuffer))?.run {
                 val drillAdminHeader = """drill-admin-url"""
-                getMethod(
+                val method = getMethod(
                     "doFilter",
                     "(Ljavax/servlet/ServletRequest;Ljavax/servlet/ServletResponse;)V"
-                )?.insertBefore(
+                )?: run{
```
Comment (LikhanovMaxim) [inline src/jvmMain/kotlin/com/epam/drill/agent/instrument/TomcatTransformer.kt:43]:
```suggestion
                )?: run {
```

**Review by LikhanovMaxim — COMMENTED** [review]
checks are failed

File: src/commonNative/kotlin/com/epam/drill/request/PluginExtension.kt [hunk 2]
```diff
@@ -9,4 +9,10 @@ actual object PluginExtension {
             it.processServerRequest()
         }
     }
+
+    actual fun processServerResponse(){
```
Comment (LikhanovMaxim) [inline src/commonNative/kotlin/com/epam/drill/request/PluginExtension.kt:13]: formatting

File: src/commonNative/kotlin/com/epam/drill/request/PluginExtension.kt [hunk 3]
```diff
@@ -9,4 +9,10 @@ actual object PluginExtension {
...
+        pstorage.values.filterIsInstance<GenericNativePlugin>().forEach {
```
Comment (LikhanovMaxim) [inline src/commonNative/kotlin/com/epam/drill/request/PluginExtension.kt:14]: refactoring: duplicate

**Review by LikhanovMaxim — COMMENTED** [review]
Check errors, add try-catch?

File: src/jvmMain/kotlin/com/epam/drill/request/PluginExtension.kt [hunk 4]
```diff
@@ -20,4 +20,6 @@ import com.epam.drill.kni.*
 @Kni
 actual object PluginExtension {
     actual external fun processServerRequest()
```
Comment (Mazel-Tovr) [inline src/jvmMain/kotlin/com/epam/drill/request/PluginExtension.kt:22]: uncomment invoke of this method after admin part of t2c will be in prod

**Pull request merged**